diff --git a/dist/itchDownloader/initializeBrowser.js b/dist/itchDownloader/initializeBrowser.js
index 7c520e7752aac6ca0427445d45c6353f4673b3ee..d965988ff723416c5a300e9cf0cc28900f0895e1 100644
--- a/dist/itchDownloader/initializeBrowser.js
+++ b/dist/itchDownloader/initializeBrowser.js
@@ -44,7 +44,6 @@ const initializeBrowser = (_a) => __awaiter(void 0, [_a], void 0, function* ({ d
             behavior: 'allow',
             downloadPath: path_1.default.resolve(downloadDirectory),
         });
-        yield client.send('Browser.enable');
         let totalBytes = 0;
         let fileName = '';
         client.on('Browser.downloadWillBegin', (event) => {
diff --git a/src/itchDownloader/initializeBrowser.ts b/src/itchDownloader/initializeBrowser.ts
index dd86199670a33cabc62d58917b95aa3289dc01f6..5fe702db865e97017351c3de8a51aea8f5e3bf27 100644
--- a/src/itchDownloader/initializeBrowser.ts
+++ b/src/itchDownloader/initializeBrowser.ts
@@ -45,7 +45,6 @@ export const initializeBrowser = async ({
       behavior: 'allow',
       downloadPath: path.resolve(downloadDirectory),
     });
-    await client.send('Browser.enable' as any);
     let totalBytes = 0;
     let fileName = '';
     client.on('Browser.downloadWillBegin', (event: any) => {
